tests/Doctrine/DBAL/Migrations/Tests/VersionTest.php

```diff
@@ -407,8 +443,8 @@ public function testDryRunCausesSqlToBeOutputViaTheOutputWriter()
         $ow       = new OutputWriter(function ($msg) use (&$messages) {
             $messages[] = trim($msg);
         });
-        $config  = new Configuration($this->getSqliteConnection(), $ow);
-        $version = new Version(
+        $config   = new Configuration($this->getSqliteConnection(), $ow);
```

Comment:
Isn't this already fixed in `master`?

Reply:
(handled by #608)

Reply:
it would be great to rebase the branch to clean the diff here